packages/rocketchat-contacts/server/index.js

```diff
@@ -94,6 +96,58 @@ Meteor.methods({
 		}
 		return provider.queryContacts(weakHashes);
 	},
+	getInviteLink() {
```

Comment:
Currently if DynamicLinks are not configured properly, or if our DynamicLink creation fails, the entire invite process fails because of these Meteor.Errors.  I wonder if we should have some fallback so that we can at least send a generic invite asking them to download and start using our app?  Otherwise we have created a dependency on Firebase to always be available, reachable, and working properly.

Reply:
Good point, how about sending the server_url as the fallback option?

Reply:
I think we would need to cover both the web client and the android case.  So maybe server url with the DM path appended to it, and the app store link for the android client?